convoy/storage.py

```diff
@@ -68,13 +68,11 @@
 _STORAGE_CONTAINERS = {
     'blob_globalresources': None,
     'blob_resourcefiles': None,
-    'blob_torrents': None,
```

Comment:
Let's keep these populated in here and instead prevent the creation of the container and table. This will allow deletes to happen for a migration path forward.

I'll sync with you to explain.